src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,53 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+type Tab = {
+  id: string,
+  title: string,
+  content: string,
+};
+
+type Props = {
+  tabs: Tab[],
+  selectedTabId: string,
+  onTabSelected: (tab: Tab) => void,
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs, selectedTabId, onTabSelected,
+}) => {
+  const isSelectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={classNames({ 'is-active': tab === isSelectedTab })}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={() => {
+                  if (isSelectedTab !== tab) {
+                    onTabSelected(tab);
+                  }
+                }}
```

Comment:
```suggestion
                onClick={() => handleTabClick(tab)}
```
Avoid to use direct function executions inside event attributes. It's hard to debug and reuse. Better move it to separate function.